content/doc/find-help/faq.md

```diff
@@ -213,5 +213,10 @@ If a VACUUM operation needs more disk that there is remaining, migrating to the
 ## Where are the backups stored?
 
 Clever Cloud store all your backups on [Cellar](https://www.clever-cloud.com/product/cellar-object-storage/). 
```

Comment:
Maybe we should explain here Cellar is an replicated object storage service, and then detail it's replication mechanism. Beware of new lines/paragraphs if not needed: maybe we should regroup sentences here.